sdk-api-src/content/winperf/nc-winperf-pm_collect_proc.md

```diff
@@ -143,9 +140,9 @@ The following tests are performed only if test level 1 is used:
 
 #### Examples
 
-For an example, see [Implementing CollectPerformanceData](/windows/desktop/PerfCtrs/implementing-collectperformancedata).
+See [Implementing CollectPerformanceData](/windows/desktop/PerfCtrs/implementing-collectperformancedata).
 
 ## -see-also
 
-- <a href="/previous-versions/windows/desktop/legacy/aa372200(v=vs.85)">OpenPerformanceData</a>
+- [OpenPerformanceData](/previous-versions/windows/desktop/legacy/aa372200)
```

Comment:
No. This is wrong, 404. The `(v=vs.85)` part is required.

I intentionally left it as an `<a>` tag because parentheses aren't friendly with markdown syntax. Then again, we need to port this last remaining article to this repository.

Reply:
ok, reverted.